cookbook/core/flyte_basics/lp.py

```diff
@@ -87,3 +97,62 @@ def go_greet(day_of_week: str, number: int, am: bool = False) -> str:
     else:
         # We're extra enthusiastic on weekends
         print(morning_greeting(number=3, day_of_week=weekday))
+
+# %%
+# Partial Inputs for Launch Plans
```

Comment:
I think this and the next section can be combined with the first one. If that's the case, you can ignore the below-suggested changes.

Reply:
@cosmicBboy

Reply:
I the purpose of these updates is to move content from the old FAQs pages to the User Guide. For reference, the changes in this PR are from this page: https://docs.flyte.org/en/latest/howto/launchplans.html

I think in this case we can remove all the stuff from `Partial Inputs for Launch Plans` and below, since it looks like there's a more comprehensive code example in the section above.

Reply:
@cosmicBboy please confirm this looks right. The deleted section will not be placed anywhere else in the docs.